.github/workflows/ci_pipeline.yml

```diff
@@ -68,7 +68,7 @@ jobs:
           secretId: ${{ secrets.VAULT_SECRET_ID }}
           secrets: |
             kv/team/text/data/harbor * ;
-            kv/team/text/data/hugin-stage DATABASE_URL_STAGE
+            kv/team/text/data/hugin-stage *;
```

Comment:
Kunne sikkert laget en egen secret i vault for keycloak greier (ikke under Hugin) siden det garantert blir brukt flere plasser i framtiden.

Reply:
Ikkje dumt, flyttet keycloak-verdier fra hugin til en egen sti i vault. Ikke helt sikker på om syntaksen for å hente flere secrets er rett tho :thinking:

Reply:
Tror det er rett, har søkt det opp før! [Her er noen eksempler](https://github.com/hashicorp/vault-action)

Reply:
Du trenger ikke semikolon på siste da, men regner med det ikke skader